lib/commands/new/project-template.js

```diff
@@ -328,7 +322,7 @@ exports.ProjectTemplate = class {
             "aurelia-templating-binding",
             {
```

Comment:
Thanks for the double check on that one. Wouldn't have hurt to have it that way, but there's no reason _to_ have it that way either :)